lang/docs/src/generate.rs

```diff
@@ -36,21 +38,18 @@ impl Generate for Dtor {
 
 impl Generate for DocComment {
     fn generate(&self) -> String {
+        let mut options = Options::default();
+        options.render.hardbreaks = true;
         let DocComment { docs } = self;
         let prefix = "<span class=\"comment\">";
         let postfix = "</span>";
-        docs.iter()
-            .map(|doc| {
-                format!(
-                    "{} {} {}",
-                    prefix,
-                    askama_escape::escape(doc, askama_escape::Html),
-                    postfix
-                )
-            })
+        let text = docs
+            .iter()
+            .map(|doc| askama_escape::escape(doc, askama_escape::Html).to_string())
```

Comment:
`askama_escape::escape` is not necessary here anymore: `comrak::markdown_to_html` scrubs HTML by default. The difference is that with `askama_escape::escape`, raw HTML is displayed (escaped), while with `comrak`, it gets removed.
I personally prefer displaying it, so my recommendation would be:

* Use [`options.render.escape = true`](https://docs.rs/comrak/latest/comrak/struct.RenderOptions.html#structfield.escape) to escape HTML using `comrak`
* Get rid of the `askama_escape` call here.